src/views/Anuncis/components/ServicesList/ServicesList.css

```diff
@@ -1,7 +1,49 @@
-.servicesCont {
+.services-cont {
     margin-top: 0;
 }
 
 .first-column:first-child {
     padding-left: 0px;
+}
+
+.service-columns {
+    margin: 0;
+    padding: 0;
+    padding-top: 5px;
+}
+
+.service-columns:last-child {
+    margin: 0;
+}
+
+.service-column {
+    padding: 0;
+    margin: 0;
+}
+
+.service-column:last-child {
+    padding-left: 15px;
+}
+
+@media (max-width: 768px) {
+    .service-column:last-child {
+        padding: 0px;
+    }
+    
+    .left-column {
+        padding-right: .5rem;
+    }
+
+    .right-column {
+        padding-left: .5rem;
+    }
+
+    .right-column>.service-column:last-child,
+    .left-column>.service-column:last-child {
+        padding-top: 1rem;
+    }
+
+    .right-column>.service-column:last-child {
+        padding-top: 1rem;
+    }
 }
```

Comment:
Aquest css m'imagino que és una puta guarrada, però només he estat capaç d'aconseguir exactament el que volia fent-ho així. 
Si us fa tant mal a la vista que no podeu suportar-ho us animo a que ho milloreu, sempre i quan es segueixi veient igual de bé. Jo no ho tocaré més...

Reply:
Si dius que no has de tocar les hipotètiques millores que et podem plantejar, fes merge a Master directament sense passar per Pull Request, i així no perdem el temps revisant el teu codi..

Reply:
Una mica radikal... He dedicat molt de temps per aconseguir que es vegi tot be lluitant contra bulma, no tocare mes aquest css en concret xo el comentari nomes anava x aquesta part, he acceptat sempre els comentaris, una miqueta de calma...